From d22f141a636aa895f392193eee17bcf6a9499222 Mon Sep 17 00:00:00 2001
From: Chris Gorman <chrisjohgorman@gmail.com>
Date: Mon, 24 Jun 2024 13:54:29 -0400
Subject: [PATCH] Using os.path.join is unnecessary as get_cofig_file_path
 returns the full path to the grc.conf file.

- With os.path.join, the file returned under windows is C:/grc.conf.
  This is not ideal, so try without it as get_config_file_path returns
  both the directory and file component of the path.
---
 grc/gui/Config.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/grc/gui/Config.py b/grc/gui/Config.py
index 91b3a331a..dac7a5772 100644
--- a/grc/gui/Config.py
+++ b/grc/gui/Config.py
@@ -28,8 +28,7 @@ class Config(CoreConfig):
 
     name = 'GNU Radio Companion'
 
-    gui_prefs_file = os.environ.get(
-        'GRC_PREFS_PATH', os.path.join(get_config_file_path(), 'grc.conf'))
+    gui_prefs_file = os.environ.get('GRC_PREFS_PATH', get_config_file_path())
 
     def __init__(self, install_prefix, *args, **kwargs):
         CoreConfig.__init__(self, *args, **kwargs)
-- 
2.45.2

